javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,74 @@
+//Find the shortest world
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function findShortestWord(array) {
+  let currentWord;
+  for (let i = 0; i < array.length; i++) {
+    if (i == 0) {
+      currentWord = array[i];
+    } else if (array[i].length < currentWord.length) {
+      currentWord = array[i];
+    }
+  }
+  return currentWord;
+}
+
+console.log(findShortestWord(danishWords));
+
+//Find and count the Danish letters
+const danishLetter = ["æ", "ø", "å"];
+
+function countDanishLetters(string) {
+  let countAe = 0;
+  let countOe = 0;
+  let countAa = 0;
+  for (let i = 0; i < string.length; i++) {
+    if (string[i] == danishLetter[0]) {
+      countAe++;
+    } else if (string[i] == danishLetter[1]) {
+      countOe++;
+    } else if (string[i] == danishLetter[2]) {
+      countAa++;
+    }
+  }
+  return (
+    "total: " +
+    (countAe + countOe + countAa) +
+    ", æ: " +
+    countAe +
+    ",ø : " +
+    countOe +
+    ", å: " +
+    countAa
+  );
+}
+
+const danishString = "Jeg har en blå bil";
+console.log(countDanishLetters(danishString)); // returns {total: 1, å: 1}
+
+const danishString2 = "Blå grød med røde bær";
+console.log(countDanishLetters(danishString2)); // returns {total: 4, æ: 1, ø: 2, å: 1}
```

Comment:
Well done! 
It is good to think about one more thing. If you need to check more letters you should write more case statements.
So you can write like this.

function countDanishLetters(danishString){
 let count = { total: 0 };
    for (let i = 0; i < danishString.length; i++) {
        if (danishLetters.includes(danishString[i])) {
            count.total = count.total + 1;
            if (count.hasOwnProperty(danishString[i])) {
                count[danishString[i]] += 1;
            } else {
                count[danishString[i]] = 1;
            }
        }
    } 
    console.log(count);
}

Here we're checking if a character is a key in the object with hasOwnProperty, which will return true if the character is missing. In that case, we initialize it to 1, otherwise, we add one to what is already there.